README.md

```diff
@@ -1,21 +1,7 @@
 # ns8-piler
 
-This is a template module for [NethServer 8](https://github.com/NethServer/ns8-core).
-To start a new module from it:
-
-1. Click on [Use this template](https://github.com/NethServer/ns8-piler/generate).
-   Name your repo with `ns8-` prefix (e.g. `ns8-mymodule`). 
-   Do not end your module name with a number, like ~~`ns8-baaad2`~~!
-
-1. An automated initialization workflow starts: wait for its completion.
-   You can follow the run inside the "Actions" tab, the workflow is named "Initial commit"
-
-1. You can now clone the repository
-
-1. Edit this `README.md` file, by replacing this section with your module
-   description
-
-1. Commit and push your local changes
+Start and configure a piler instance.
+The module uses [piler Docker Image](https://hub.docker.com/r/sutoj/piler).
```

Comment:
Could you provide a link to the container source code and to Piler source code? 

I had a glance to https://mailpiler.com/ and I can't even find code license information.

Reply:
please check https://bitbucket.org/jsuto/piler/